docs/pcb_basics.md

```diff
@@ -0,0 +1,76 @@
+## PCB Basics and Troubleshooting
+![](images/pcb_basics/pcb_angle_crop.jpg)
+
+  
+
+## PCB Basics
+
+Building a keyboard for the first time can be overwhelming. In this guide, we are going to go over the basics of what is generally considered to be the most complex part of a keyboard kit: The PCB.
+
+## Overview
+There are a handful of major components to a PCB. Not all PCBs will have each of these.
+
+ -   Switch cutouts
+	 - One large cutout for the middle of the switch.
+	 - Two smaller cutouts for alignment pins.
+	 - Two small cutouts for the metal pins.
+ - LED cutouts
+	 - One negative, and one positive hole, where LEDs can be
+   soldered in.
+		-	2 holes/contacts are needed for single color LEDs.
+		-	3 holes/contacts are needed for RGB LEDs.

```

Comment:
i would remove this, since none of our PCBs will have RGB LEDs like this (only SMD) and we should focus on ours.
also i think most RGB LEDs have 4 pins.